uberfire-extensions/uberfire-wires/uberfire-wires-core/uberfire-wires-core-grids/src/main/java/org/uberfire/ext/wires/core/grids/client/widget/grid/impl/BaseKeyboardOperation.java

```diff
@@ -98,16 +102,14 @@ private boolean isHeaderSelectionOriginSet(final GridWidget selectedGridWidget)
         return !gridModel.getSelectedHeaderCells().isEmpty();
     }
 
-    private boolean isGridWidgetRendered(final GridWidget gridWidget) {
-        final BaseGridRendererHelper rendererHelper = gridWidget.getRendererHelper();
-        final BaseGridRendererHelper.RenderingInformation renderingInformation = rendererHelper.getRenderingInformation();
+    private boolean isGridWidgetRendered(final BaseGridRendererHelper.RenderingInformation renderingInformation) {
         return renderingInformation != null;
     }
 
-    private boolean isGridColumnCandidateForScroll(final GridWidget gridWidget, final boolean isHeaderCellSelected) {
+    private boolean isGridColumnCandidateForScroll(final GridWidget gridWidget,
+                                                   final BaseGridRendererHelper.RenderingInformation renderingInformation,
+                                                   final boolean isHeaderCellSelected) {
         final GridData gridModel = gridWidget.getModel();
```

Comment:
You could move this line to after the check for `renderingInformation == null`.

.. and why are you not using `Objects.isNull(..)`!!!! (You're the main promoter of it's use!)

Reply:
nope, I am promoter of `equals`, `isNull`  I asked just in case class mixed `isNull` and `== null`

Reply:
Indeed in if statement  Objects.isNull(..) could be used, however it was made mainly for lambda expressions. Also some people prefer to use ligatures. They work with == sign.